### PR TITLE
Demo changes - tightening up UI

### PR DESCRIPTION
**Pull request by mmichalek**
### Summary
Tighten up UI for NRF.  Change default font to Lato.

### CODE REVIEW

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/home/home.component.scss [hunk 1]
```diff
@@ -13,9 +13,9 @@
   border-radius: 4px;
 
   &.tablet {
-    margin: 15px;
-    padding: 15px;
-  }
+    margin: 18px;
```
Comment (jchirico8) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/home/home.component.scss:16]: hmm "tightening" up ui by increasing the spaces 🤔

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/instructions/instructions.component.html [hunk 2]
```diff
@@ -1,2 +1,2 @@
-<div *ngIf="instructions" class="instructions-color" [ngClass]="instructionsSize" responsive-class [innerHTML]="instructions | markdownFormatter">
+<div *ngIf="instructions" class="muted-color" [ngClass]="instructionsSize" responsive-class [innerHTML]="instructions | markdownFormatter">
```
Comment (stojankz) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/instructions/instructions.component.html:1]: hm what's the reasoning for changing this color? instructions-color uses the secondary-text whereas muted-color uses disabled-text. secondary-text seems like the right theme color to use here.. if we wanted the secondary-text theme color to be lighter then I think the more accurate solution would be to update _app-theme?

**Review by jchirico8 — DISMISSED** [review]

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/primary-button/_primary-button-theme.scss [hunk 3]
```diff
@@ -10,6 +10,7 @@
             max-width: 100%;
             &[disabled] {
                 border-color: $disabled-border;
```
Comment (stojankz) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/primary-button/_primary-button-theme.scss:12]: we don't need this border-color if you're setting border-style to none

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/bacon-strip/bacon-strip.component.html [hunk 4]
```diff
@@ -1,5 +1,6 @@
 <div class="container mat-elevation-z5" responsive-class>
     <div class="leftside" [class.expanded]="searchExpanded">
+        <input cdkFocusInitial mat-icon-button type='text' class="hidden-focus-button"/>
```
Comment (stojankz) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/bacon-strip/bacon-strip.component.html:3]: what is this input used for?

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/home/home.component.scss (hunk 1 above)
Comment (mmichalek) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/screens-with-parts/home/home.component.scss:16]: 😛

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/instructions/instructions.component.html (hunk 2 above)
Comment (mmichalek) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/instructions/instructions.component.html:1]: This was to match other grey text on the sales screen, so to overall reduce the number of different greys on the sale screen.

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/bacon-strip/bacon-strip.component.html (hunk 4 above)
Comment (mmichalek) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/screen-parts/bacon-strip/bacon-strip.component.html:3]: This input in a workaround to hide the initial keyboard focus.  it is a fix for the little shadow circle that would always show up behind the home icon on the home screen.  I spent too long looking for a better solution. :)

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/primary-button/_primary-button-theme.scss (hunk 3 above)
Comment (mmichalek) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/shared/components/primary-button/_primary-button-theme.scss]: Alright.

**Review by stojankz — APPROVED** [review]

**Pull request merged**